homeassistant/components/nexia/strings.json

```diff
@@ -64,7 +64,7 @@
   "services": {
     "set_aircleaner_mode": {
       "name": "Set air cleaner mode",
-      "description": "The air cleaner mode.",
+      "description": "Sets the air cleaner mode of the thermostat.",
```

Comment:
Not sure why "of the thermostat" is needed here. Same for other places.

Reply:
The integration is for thermostats, but no problem: I've removed all three.

The main point was getting the descriptions corrected.